pyproject.toml

```
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools >= 36.7.0", "wheel >= 0.23.0"]
```

`long_description_content_type` is only supported since setuptools 38.6.0.

```suggestion
requires = ["setuptools >= 38.6.0", "wheel >= 0.23.0"]
```

---

@KOLANICH This was _not_ resolved in [`bfacaec...5858f14`](https://github.com/kaitai-io/kaitai_struct_python_runtime/compare/bfacaec50a0386ffa3d768ec701a3ca923f41068..5858f1401a8c3581b91e0daff4367c7d66604507) - you relaxed the `setuptools` version requirement to <code>&gt;= 3<b><ins>6</ins></b>.6.0</code> but it should have been **increased** to <code>&gt;= 3<b><ins>8</ins></b>.6.0</code>.